llvm/lib/Target/X86/X86ExpandPseudo.cpp

```diff
@@ -613,6 +613,87 @@ bool X86ExpandPseudo::expandMI(MachineBasicBlock &MBB,
   case X86::CALL64m_RVMARKER:
     expandCALL_RVMARKER(MBB, MBBI);
     return true;
+  case X86::ADD32mi_ND:
```

Comment:
Add a comment we intend take them as pseudo instructions.

Reply:
Done